lib/unhangout-routes.js

```diff
@@ -491,18 +549,51 @@ module.exports = {
             showEventOverview(req, res, "admin.ejs");
         });
 
-        app.get("/myevents/", ensureAuthenticated, function(req, res) {
+        app.get("/myevents/", ensureAuthenticated, utils.ensureAdmin,function(req, res) {
             showEventOverview(req, res, "myevents.ejs");
         });
 
+        app.post("/myevents/", ensureAuthenticated, utils.ensureAdmin, function(req, res) {
+
+            var eventID = req.body.eventId; 
+
+            var event = db.events.get(req.body.eventId);
```

Comment:
This doesn't verify that the current user is an admin of this event.  ``utils.ensureAdmin`` only verifies that the user is an admin of some event somewhere, not that they are an admin of the event referenced by ``res.body.eventId``.  So in effect this would let anyone edit admin-ship of any event if they have ever been an admin (superuser-like capability).

It should check that the user is an admin of this event before letting them continue.